ios/samples/KMSample2/build.sh

```diff
@@ -29,17 +29,11 @@ assert ( ) {
 
 verify_KMEI ( ) {
     KMEI_BUILD_EXISTS=true
-    if ! [ -d "${LIB_KEYMAN_SRC}/include" ]; then
-        KMEI_BUILD_EXISTS=false
-    elif ! [ -d "${LIB_KEYMAN_SRC}/Keyman.bundle" ]; then
-        KMEI_BUILD_EXISTS=false
-    elif ! [ -f "${LIB_KEYMAN_SRC}/libKeyman.a" ]; then
-        KMEI_BUILD_EXISTS=false
-    fi
+    [ -d "$LIB_KEYMAN_SRC" ] || KMEI_BUILD_EXISTS=false
 }
 
-LIB_KEYMAN_SRC=../../engine/KMEI/build/libKeyman
-LIB_KEYMAN_DEST=./
+LIB_KEYMAN_SRC=../../engine/KMEI/build/Debug-universal/KeymanEngine.framework
```

Comment:
Wait... _Debug_-universal - exclusively?  Not release?  Why?  Shouldn't that at least be configurable via command-line parameter?